### PR TITLE
update Renegade X CDNs

### PR DESCRIPTION
**Pull request by matthewh86**
### What CDN does this PR relate to
Renegade X

### Does this require running via sniproxy
No

### Capture method
Charles Proxy

### Testing Scenario
Home LAN

### Testing Configuration
default docker-compose from the lancache repo

### Sniproxy output
Please paste the output from `docker logs <sniproxy container name/id> | sed 's/.*\:443 \[//;s/\].*//' | sort | uniq -c` below
```
n/a
```

### CODE REVIEW

**Comment by GotenXiao** [comment]
CC @Cronus89

**Comment by Cronus89** [comment]
Looks fine except remove
static.renegade-x.com
This should never be cached.

There are more changes, but I'll send those as a separate PR myself later. This gets the new servers in the list however :)

File: renegadex.txt [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 mirror.us.tyrant.gg
 mirror.usa.tyrant.gg
 rxp-de1.ts3-server.ch
 rxp-lax2.cncfps.com
+apac-sg.nexusbytes.ren-x.com
+apac-tyo.nexusbytes.ren-x.com
+eu-uk.nexusbytes.ren-x.com
+eu-nl.nexusbytes.ren-x.com
+us-mia.buyvm.ren-x.com
+static.renegade-x.com
```
Comment (MathewBurnett) [inline renegadex.txt:14]: - [ ] remove static.renegade-x.com

**Review by MathewBurnett — CHANGES_REQUESTED** [review]
Just that static content to remove and then this looks fine

**Review by mintopia — APPROVED** [review]

**Comment by mintopia** [comment]
Thanks for the PR!

**Pull request merged**